pkg/vm/create_evm.go

```diff
@@ -121,29 +113,33 @@ func CreateEVMGenesis(
 		)
 	}
 
-	getPrecompiles(conf, params, &genesis.Timestamp)
+	precompiles := getPrecompiles(params, utils.TimeToNewUint64(time.Now()))
```

Comment:
can we use a variable for genesis block 0 timestamp and set it to utils.TimeToNewUint64(time.Now()),
then pass it to getPrecompiles and also as a new field of subnetSDK.SubnetParams so all timestamps can
be specified (and in this case, genesis block 0 timestamp to equal precompile timestamps)